--- config.def.h
+++ config.h
@@ -201,8 +201,10 @@
 	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
 	{ ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
 	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
-	{ ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },
-	{ ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} },
+	{ ShiftMask,            XK_Up,          kscrollup,      {.i =  3} },
+	{ ShiftMask,            XK_Down,        kscrolldown,    {.i =  3} },
+	{ ControlMask|ShiftMask,XK_e,           zoom,           {.f = +1} },
+	{ ControlMask|ShiftMask,XK_q,           zoom,           {.f = -1} },
 };
 
 /*
